content/vendor/snapshots/restores.md

```diff
@@ -0,0 +1,16 @@
+---
+date: 2019-01-23
+linktitle: "Restores"
+title: Restores
+aliases:
+  - /vendor/snapshots/
+---
+
+Click Restore on a snapshot on the `/apps/<slug>/snapshots` page to click a restore.
+
+First kotsadm will undeploy the application version by running `kubectl delete --wait` on the release yaml.
```

Comment:
I'm a bit lost here. I thought we were in Vendor? If this is in the Admin Console, I'd imagine this would go in that section. Something like: Admin Console -> Backup and Recovery -> Restoring from a Snapshot